explainer.md

```diff
@@ -279,21 +293,19 @@ CSS can be used to make changes to the default animation, or override `animation
 
 ### JavaScript access to elements
 
-So far the transition elements have been addressed by pseudo-element selectors, but JavaScript could be given access to the elements. This would be done via a shadow root.
-
-This isn't unusual, as developer currently use things like [`::placeholder`](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder) to address things in a UA shadow root within `<input>`. We can't expose the shadow root within `<input>`, because the structure is non-standard, however, for transitions, the structure will be standardized, so it can be exposed.
+So far the transition elements have been addressed by pseudo-element selectors, but JavaScript could be given access to the elements using the [CSSPseudoElement](https://drafts.csswg.org/css-pseudo-4/#CSSPseudoElement-interface) interface.
 
 ```js
 // In Page-B
 document.performTransition((transition) => {
   // Build up transition parts, then…
-  transition.root.querySelector('[part=header]').animate(…);
+  document.documentElement.pseudo("page-transition-container(name)").animate(...);
 });
 ```
 
 - [Open question](https://github.com/WICG/shared-element-transitions/issues/86) What's the deadline for calling `performTransition`?
 
-If the 'stage' of the transition is exposed as a shadow root like this, the developer can interact with the elements in a regular way. The developer could even create elements specifically for the transition.
+Note that using pseudo-elements here implies that the developer can not change the DOM structure of transition elements. An alternate approach considered to provide this extensibility uses shadow DOM. The pseudo-element option was preferred to make it easier to build on concepts which already exist in the platform and ensure ease of implementation. See [issue](https://github.com/WICG/shared-element-transitions/issues/93) for detailed discussion.
```

Comment:
This feels like it's saying shadow DOM doesn't already exist in the platform.

Reply:
The concept that's missing is creating a shadow DOM which has no element to be its shadow host.